xonsh/webconfig/elm-compile.xsh

```diff
@@ -199,17 +199,23 @@ type alias XontribData =
 xontribs : List XontribData
 xontribs ="""
 
+
+def get_xontrib_item(xontrib_name:str, xontrib:Xontrib):
```

Comment:
Should there be spaces between the colons and the types here?

Reply:
updated.